### ISSUE TITLE
Can't download fonts from the website at the moment?

### ISSUE THREAD

**Issue by tobychidi** [DecliningLotus/fontsource#736]
### Describe the bug

https://api.fontsource.org/v1/fonts/playfair-display/download  for example returns 
```
{
    "statusCode": 500,
    "message": "Internal server error"
}
```

### Steps to Reproduce

1. Go to fontsource.org
2. Pick a font you like
3. Make sure it looks good 😅
4. Click download to get the desktop fonts
5. Get disappointed 😂

### Expected behavior

1. Go to fontsource.org
2. Pick a font you like
3. Make sure it looks good 😅
4. Click download to get the desktop fonts
5. Install and enjoy your new fonts

### OS

Windows

### Browser

Edge

**Comment by tobychidi** [comment]
New error:

```
{
    "statusCode": 404,
    "message": "Not found: crimson-pro/download",
    "error": "Not Found"
}
```

**Comment by ayuhito** [comment]
Resolved in #734. Although the new URL on the website uses: https://api.fontsource.org/v1/download/playfair-display

**Comment by tobychidi** [comment]
Can confirm. Works well. Thank you
